samples/msal-react-samples/react-router-sample/src/authConfig.js

```diff
@@ -1,9 +1,22 @@
+// Browser check variables
+// If you support IE, our recommendation is that you sign-in using Redirect APIs
+// If you as a developer are testing using Edge InPrivate mode, please add "isEdge" to the if check
+const ua = window.navigator.userAgent;
+const msie = ua.indexOf("MSIE ");
+const msie11 = ua.indexOf("Trident/");
+const msedge = ua.indexOf("Edge/");
+const isIE = msie > 0 || msie11 > 0;
+const isEdge = msedge > 0;
+
 // Config object to be passed to Msal on creation
 export const msalConfig = {
     auth: {
         clientId: "0a61c279-646b-4055-a5f1-1c3da7f70f18",
         redirectUri: "/",
         postLogoutRedirectUri: "/"
+    },
+    cache: {
+        storeAuthStateInCookie: isIE || isEdge
```

Comment:
Firefox is another browser this is recommended for (for incognito where local storage can be cleared when redirecting)

Reply:
Done